api/serializers.py

```diff
@@ -12,6 +12,15 @@
 class ProfileSerializer(serializers.ModelSerializer):
     def to_representation(self, instance):
         rep = super().to_representation(instance)
+
+        # Replace image file path with the file data.
+        image = instance.image
+        if hasattr(image, "file"):
```

Comment:
This raises a ValueError if there is no file

Reply:
I'm having trouble reproducing this. Could you summarize the steps you took to cause this throw a value error?

Reply:
It throws `ValueError: The 'image' attribute has no file associated with it.` when no image has been saved to it yet

Reply:
is there anything wrong with using `image != ""` instead

Reply:
That was not working for me which is why I suggested the change